### ISSUE TITLE
Question about SemanticKITTI and PTv3 Performance

### ISSUE THREAD

**Issue by xiaosa269** [Pointcept/PointTransformerV3#73]
Thank you for your work. Could you please provide the mIoU for each individual category in SemanticKITTI? Do you think PTv3 performs poorly in segmenting smaller objects?

**Comment by Gofinge** [comment]
Hi, due to a change of corporation organization. I can not access the original training log anymore. But I can retrain it recently.

**Comment by xiaosa269** [comment]
Thanks

**Comment by xiaosa269** [comment]
@Gofinge Hello, sorry to bother you again. After running sh scripts/test.sh -p python -g 2 -d semantic_kitti -n semantic_v3_test_val_epoch50 -w model_best, I noticed that the test program is still testing on the val dataset. What command should I run to validate on the test dataset?
![image](https://github.com/user-attachments/assets/696d936f-964a-4a40-9710-a93be158a8bf)


The above image shows the way I have divided my own dataset. I noticed that after running the test script, it is still testing on dataset 3. Thank you very much for answering my question.
